docrepr/__init__.py

```diff
@@ -24,3 +26,8 @@
     'use_qt4': False,
     'outline': False
 }
+
+# This imports IPython and PySide/PyQt4
+# [ ] find a way to do delayed import
+from .webview import webview

```

Comment:
This adds PyQt4/PySide as a dependency to `docrepr`, right? If that's the case, I'd like to see a different approach because I don't want `docrepr` to depend on any of them.